src/test/java/co/rsk/federate/bitcoin/BitcoinTestUtils.java

```diff
@@ -0,0 +1,28 @@
+package co.rsk.federate.bitcoin;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.Sha256Hash;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BitcoinTestUtils {
```

Comment:
we might improve this utility class by putting the `final` keyword at level class.

Reply:
Done